src/test/data/draft/createDraft.ts

```diff
@@ -0,0 +1,118 @@
+import { individualDetails } from './partyDetails'
```

Comment:
File name `src/test/data/draft/createDraft.ts` is wrong, you do not export `createDraft` function here, you export sample JSON object. Please rename to something like `claimDraft`.

There is also `data/draft/claimDraft.ts` already there, any reason you did not decide to use that one?

REQ